group_vars/all.yml.sample

```diff
@@ -271,7 +271,7 @@ dummy:
 #osd_mkfs_type: xfs
```

Comment:
Instead of editing this file you should edit `roles/ceph-osd/defaults/main.yml` and then run `generate_group_vars_sample.sh`. Same for `roles/ceph-common/defaults/main.yml`.

Please also note that the option `enable experimental unrecoverable data corrupting features: 'bluestore rocksdb'` won't stay forever so we should flag this by saying something like "only valid for Kraken, might go away for Luminous".